`beamPackages.rebar3Relx` appears to be broken

### Describe the bug
When building a package with `releaseType = "release"` instead of `releaseType = "erscript"` `rebar3Relx` fails in the postInstall phase:
```
@nix { "action": "setPhase", "phase": "installPhase" }
installing
ERTS found in /nix/store/8230bn8m34ns4mjz88c8qq9la0q2k9gp-eturnal-1.12.0/rel/eturnal/erts-13.2.2.3 - removing references to erlang to reduce closure size
consumeEntire(): ERROR: Input null bytes, won't process
/nix/store/3pfjacvdg9f491lfqc9qb2d0nknx73fb-stdenv-linux/setup: line 144: pop_var_context: head of shell_variables not a function context
```

<details>
<summary>Built derivation</summary>

```nix
{ lib, beamPackages, fetchFromGitHub, libyaml }:
beamPackages.rebar3Relx rec {
  pname = "eturnal";
  version = "1.12.0";

  src = fetchFromGitHub {
    owner = "processone";
    repo = "eturnal";
    rev = version;
    sha256 = "BcFFsdPJ/zxs0iPvfiS0P801BqdxveL0hayc2Yld6qs=";
  };

  checkouts = beamPackages.fetchRebar3Deps {
    inherit version src;
    name = pname;
    sha256 = "stPusgmwNFRp2n4W6VehTmEejtgOWNblH3Z9nHQ0o8c=";
  };

  buildInputs = [ libyaml.dev ];
  buildPlugins = [ beamPackages.pc ];
  profile = "prod";
  releaseType = "release";

  meta = with lib; {
    homepage = "https://eturnal.net/";
    description = "standalone STUN / TURN server";
    license = licenses.asl20;
    platforms = platforms.all;
  };
}

```

</details>

<details>
<summary>Full build log</summary>

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/g1m91q6a7cb90kngg17pdrpyr57ign55-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
@nix { "action": "setPhase", "phase": "buildPhase" }
building
===> Evaluating config script "rebar.config.script"
===> 25.3.2.6 satisfies the requirement for minimum OTP version 21.0
===> error undef [{rebar3_lint,module_info,[exports],[]},
                         {rebar_plugins,validate_plugin,1,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,137}]},
                         {rebar_plugins,handle_plugin,5,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,114}]},
                         {rebar_plugins,'-handle_plugins/4-fun-0-',5,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,103}]},
                         {lists,foldl,3,[{file,"lists.erl"},{line,1350}]},
                         {rebar_plugins,handle_plugins,4,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,102}]},
                         {lists,foldl,3,[{file,"lists.erl"},{line,1350}]},
                         {rebar3,run_aux,2,
                             [{file,"/build/source/apps/rebar/src/rebar3.erl"},
                              {line,181}]}]
===> Errors loading plugin rebar3_lint. Run rebar3 with DEBUG=1 set to see errors.
===> Expanded command sequence to be run: [as]
===> Running provider: as
===> Expanded command sequence to be run: [app_discovery,install_deps,lock,compile,release]
===> Running provider: app_discovery
===> Found top-level apps: [eturnal]
	using config: [{src_dirs,["src"]},{lib_dirs,["apps/*","lib/*","."]}]
===> Evaluating config script "/build/source/_checkouts/stun/rebar.config.script"
===> 25.3.2.6 satisfies the requirement for minimum OTP version 21.0
===> error undef [{rebar3_lint,module_info,[exports],[]},
                         {rebar_plugins,validate_plugin,1,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,137}]},
                         {rebar_plugins,handle_plugin,5,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,114}]},
                         {rebar_plugins,'-handle_plugins/4-fun-0-',5,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,103}]},
                         {lists,foldl,3,[{file,"lists.erl"},{line,1350}]},
                         {rebar_plugins,handle_plugins,4,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,102}]},
                         {lists,foldl,3,[{file,"lists.erl"},{line,1350}]},
                         {rebar_prv_app_discovery,do,1,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_prv_app_discovery.erl"},
                              {line,39}]}]
===> Errors loading plugin rebar3_lint. Run rebar3 with DEBUG=1 set to see errors.
===> Running provider: install_deps
===> Verifying dependencies...
===> Evaluating config script "/build/source/_checkouts/fast_yaml/rebar.config.script"
===> error undef [{rebar3_hex,module_info,[exports],[]},
                         {rebar_plugins,validate_plugin,1,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,137}]},
                         {rebar_plugins,handle_plugin,5,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,114}]},
                         {rebar_plugins,'-handle_plugins/4-fun-0-',5,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,103}]},
                         {lists,foldl,3,[{file,"lists.erl"},{line,1350}]},
                         {rebar_plugins,handle_plugins,4,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,102}]},
                         {lists,foldl,3,[{file,"lists.erl"},{line,1350}]},
                         {rebar_plugins,install,2,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,70}]}]
===> Errors loading plugin rebar3_hex. Run rebar3 with DEBUG=1 set to see errors.
===> Evaluating config script "/build/source/_checkouts/fast_tls/rebar.config.script"
===> Evaluating config script "/build/source/_checkouts/p1_utils/rebar.config.script"
===> Evaluating config script "/build/source/_checkouts/p1_utils/rebar.config.script"
===> Not adding provider pc compile from module pc_prv_compile because it already exists from module pc_prv_compile
===> Not adding provider pc clean from module pc_prv_clean because it already exists from module pc_prv_clean
===> Evaluating config script "/build/source/_checkouts/p1_utils/rebar.config.script"
===> Not adding provider pc compile from module pc_prv_compile because it already exists from module pc_prv_compile
===> Not adding provider pc clean from module pc_prv_clean because it already exists from module pc_prv_clean
===> 25.3.2.6 satisfies the requirement for minimum OTP version 21.0
===> error undef [{rebar3_hex,module_info,[exports],[]},
                         {rebar_plugins,validate_plugin,1,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,137}]},
                         {rebar_plugins,handle_plugin,5,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,114}]},
                         {rebar_plugins,'-handle_plugins/4-fun-0-',5,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,103}]},
                         {lists,foldl,3,[{file,"lists.erl"},{line,1350}]},
                         {rebar_plugins,handle_plugins,4,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,102}]},
                         {lists,foldl,3,[{file,"lists.erl"},{line,1350}]},
                         {rebar_plugins,install,2,
                             [{file,
                                  "/build/source/apps/rebar/src/rebar_plugins.erl"},
                              {line,70}]}]
===> Errors loading plugin rebar3_hex. Run rebar3 with DEBUG=1 set to see errors.
===> Running provider: lock
===> Running provider: compile
===> Compile (apps)
===> Running hooks for compile in app recon (/build/source/_checkouts/recon) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app quantile_estimator (/build/source/_checkouts/quantile_estimator) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app prometheus (/build/source/_checkouts/prometheus) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app accept (/build/source/_checkouts/accept) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app prometheus_httpd (/build/source/_checkouts/prometheus_httpd) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app ulitos (/build/source/_checkouts/ulitos) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app poolboy (/build/source/_checkouts/poolboy) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app influx_udp (/build/source/_checkouts/influx_udp) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app yval (/build/source/_checkouts/yval) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app p1_utils (/build/source/_checkouts/p1_utils) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app fast_tls (/build/source/_checkouts/fast_tls) with configuration:
===> 	{provider_hooks, [{pre, [{pc,compile}]}]}.
===> Running provider: {pc,compile}
===> Running hooks for {pc,compile} with configuration:
===> 	{provider_hooks, []}.
===> 	{pre_hooks, []}.
===> Compiling c_src/fast_tls.c
===> c_src/fast_tls.c: In function 'set_fips_mode_nif':
c_src/fast_tls.c:1419:2: warning: #warning OpenSSL 3 FIPS support not implemented [-Wcpp]
 1419 | #warning OpenSSL 3 FIPS support not implemented
      |  ^~~~~~~
c_src/fast_tls.c:1400:7: warning: unused variable 'ret' [-Wunused-variable]
 1400 |   int ret = 1;
      |       ^~~
c_src/fast_tls.c: In function 'get_fips_mode_nif':
c_src/fast_tls.c:1432:2: warning: #warning OpenSSL 3 FIPS support not implemented [-Wcpp]
 1432 | #warning OpenSSL 3 FIPS support not implemented
      |  ^~~~~~~

===> Compiling c_src/ioqueue.c
===>
===> Compiling c_src/p1_sha.c
===>
===> Linking /build/source/_checkouts/fast_tls/priv/lib/fast_tls.so
===> Linking /build/source/_checkouts/fast_tls/priv/lib/p1_sha.so
===> Running hooks for {pc,compile} with configuration:
===> 	{post_hooks, []}.
===> 	{pre_hooks, []}.
===> Running hooks for compile in app stun (/build/source/_checkouts/stun) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for compile in app fast_yaml (/build/source/_checkouts/fast_yaml) with configuration:
===> 	{provider_hooks, [{pre, [{pc,compile}]}]}.
===> Running provider: {pc,compile}
===> Running hooks for {pc,compile} with configuration:
===> 	{provider_hooks, []}.
===> 	{pre_hooks, []}.
===> Compiling c_src/fast_yaml.c
===>
===> Linking /build/source/_checkouts/fast_yaml/priv/lib/fast_yaml.so
===> Running hooks for {pc,compile} with configuration:
===> 	{post_hooks, []}.
===> 	{pre_hooks, []}.
===> Running hooks for compile in app conf (/build/source/_checkouts/conf) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app recon (/build/source/_checkouts/recon) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app quantile_estimator (/build/source/_checkouts/quantile_estimator) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app prometheus (/build/source/_checkouts/prometheus) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app accept (/build/source/_checkouts/accept) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app prometheus_httpd (/build/source/_checkouts/prometheus_httpd) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app ulitos (/build/source/_checkouts/ulitos) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app poolboy (/build/source/_checkouts/poolboy) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app influx_udp (/build/source/_checkouts/influx_udp) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app yval (/build/source/_checkouts/yval) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app p1_utils (/build/source/_checkouts/p1_utils) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app fast_tls (/build/source/_checkouts/fast_tls) with configuration:
===> 	{provider_hooks, []}.
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app stun (/build/source/_checkouts/stun) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app fast_yaml (/build/source/_checkouts/fast_yaml) with configuration:
===> 	{provider_hooks, []}.
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app conf (/build/source/_checkouts/conf) with configuration:
===> 	{pre_hooks, []}.
===> Analyzing applications...
===> Compiling recon
===> compile options: {erl_opts, [debug_info]}.
===> files to analyze ["/build/source/_checkouts/recon/src/recon_map.erl",
                              "/build/source/_checkouts/recon/src/recon_rec.erl",
                              "/build/source/_checkouts/recon/src/recon_trace.erl",
                              "/build/source/_checkouts/recon/src/recon_alloc.erl",
                              "/build/source/_checkouts/recon/src/recon_lib.erl",
                              "/build/source/_checkouts/recon/src/recon.erl"]
===>      Compiled recon_map.erl
===>      Compiled recon_rec.erl
===>      Compiled recon.erl
===>      Compiled recon_lib.erl
===>      Compiled recon_trace.erl
===>      Compiled recon_alloc.erl
===> Compiling quantile_estimator
===> compile options: {erl_opts, [{no_auto_import,[{ceil,1},{floor,1}]},
                                         debug_info]}.
===> files to analyze ["/build/source/_checkouts/quantile_estimator/src/quantile.erl",
                              "/build/source/_checkouts/quantile_estimator/src/quantile_estimator.erl"]
===>      Compiled quantile.erl
===>      Compiled quantile_estimator.erl
===> Compiling prometheus
===> compile options: {erl_opts, [debug_info,warn_unused_vars,
                                         warnings_as_errors,warn_export_all,
                                         warn_shadow_vars,warn_unused_import,
                                         warn_unused_function,warn_bif_clash,
                                         warn_unused_record,
                                         warn_deprecated_function,
                                         warn_obsolete_guard,
                                         strict_validation,warn_export_vars,
                                         warn_exported_vars,
                                         {d,recent_otp}]}.
===> files to analyze ["/build/source/_checkouts/prometheus/src/prometheus.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_sup.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_buckets.erl",
                              "/build/source/_checkouts/prometheus/src/contrib/prometheus_http.erl",
                              "/build/source/_checkouts/prometheus/src/contrib/prometheus_test_instrumenter.erl",
                              "/build/source/_checkouts/prometheus/src/contrib/prometheus_mnesia.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_format.erl",
                              "/build/source/_checkouts/prometheus/src/metrics/prometheus_quantile_summary.erl",
                              "/build/source/_checkouts/prometheus/src/metrics/prometheus_gauge.erl",
                              "/build/source/_checkouts/prometheus/src/metrics/prometheus_boolean.erl",
                              "/build/source/_checkouts/prometheus/src/metrics/prometheus_counter.erl",
                              "/build/source/_checkouts/prometheus/src/metrics/prometheus_histogram.erl",
                              "/build/source/_checkouts/prometheus/src/metrics/prometheus_summary.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_metric.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_time.erl",
                              "/build/source/_checkouts/prometheus/src/model/prometheus_model_helpers.erl",
                              "/build/source/_checkouts/prometheus/src/model/prometheus_model.erl",
                              "/build/source/_checkouts/prometheus/src/collectors/mnesia/prometheus_mnesia_collector.erl",
                              "/build/source/_checkouts/prometheus/src/collectors/vm/prometheus_vm_dist_collector.erl",
                              "/build/source/_checkouts/prometheus/src/collectors/vm/prometheus_vm_system_info_collector.erl",
                              "/build/source/_checkouts/prometheus/src/collectors/vm/prometheus_vm_memory_collector.erl",
                              "/build/source/_checkouts/prometheus/src/collectors/vm/prometheus_vm_statistics_collector.erl",
                              "/build/source/_checkouts/prometheus/src/collectors/vm/prometheus_vm_msacc_collector.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_registry.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_instrumenter.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_metric_spec.erl",
                              "/build/source/_checkouts/prometheus/src/formats/prometheus_protobuf_format.erl",
                              "/build/source/_checkouts/prometheus/src/formats/prometheus_text_format.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_collector.erl",
                              "/build/source/_checkouts/prometheus/src/prometheus_misc.erl"]
===>      Compiled prometheus_metric.erl
===>      Compiled prometheus_collector.erl
===>      Compiled prometheus_format.erl
===>      Compiled prometheus_instrumenter.erl
===>      Compiled prometheus.erl
===>      Compiled prometheus_http.erl
===>      Compiled prometheus_test_instrumenter.erl
===>      Compiled prometheus_mnesia.erl
===>      Compiled prometheus_time.erl
===>      Compiled prometheus_sup.erl
===>      Compiled prometheus_buckets.erl
===>      Compiled prometheus_metric_spec.erl
===>      Compiled prometheus_registry.erl
===>      Compiled prometheus_vm_msacc_collector.erl
===>      Compiled prometheus_protobuf_format.erl
===>      Compiled prometheus_misc.erl
===>      Compiled prometheus_mnesia_collector.erl
===>      Compiled prometheus_vm_memory_collector.erl
===>      Compiled prometheus_model_helpers.erl
===>      Compiled prometheus_boolean.erl
===>      Compiled prometheus_vm_system_info_collector.erl
===>      Compiled prometheus_counter.erl
===>      Compiled prometheus_vm_statistics_collector.erl
===>      Compiled prometheus_text_format.erl
===>      Compiled prometheus_gauge.erl
===>      Compiled prometheus_summary.erl
===>      Compiled prometheus_vm_dist_collector.erl
===>      Compiled prometheus_histogram.erl
===>      Compiled prometheus_quantile_summary.erl
===>      Compiled prometheus_model.erl
===> Compiling accept
===> compile options: {erl_opts, [debug_info]}.
===> files to analyze ["/build/source/_checkouts/accept/src/accept_parser.erl",
                              "/build/source/_checkouts/accept/src/accept_header.erl",
                              "/build/source/_checkouts/accept/src/accept_encoding_header.erl",
                              "/build/source/_checkouts/accept/src/accept_neg.erl"]
===>      Compiled accept_neg.erl
===>      Compiled accept_parser.erl
===>      Compiled accept_encoding_header.erl
===>      Compiled accept_header.erl
===> Compiling prometheus_httpd
===> compile options: {erl_opts, [debug_info]}.
===> files to analyze ["/build/source/_checkouts/prometheus_httpd/src/prometheus_http_config.erl",
                              "/build/source/_checkouts/prometheus_httpd/src/prometheus_httpd_ct.erl",
                              "/build/source/_checkouts/prometheus_httpd/src/prometheus_http_impl.erl",
                              "/build/source/_checkouts/prometheus_httpd/src/prometheus_httpd.erl"]
===>      Compiled prometheus_httpd.erl
===>      Compiled prometheus_http_config.erl
===>      Compiled prometheus_http_impl.erl
===>      Compiled prometheus_httpd_ct.erl
===> Compiling ulitos
===> compile options: {erl_opts, [debug_info]}.
===> files to analyze ["/build/source/_checkouts/ulitos/src/ulitos_app.erl",
                              "/build/source/_checkouts/ulitos/src/ulitos_file.erl",
                              "/build/source/_checkouts/ulitos/src/ulitos.erl"]
===>      Compiled ulitos.erl
===>      Compiled ulitos_file.erl
===>      Compiled ulitos_app.erl
===> Compiling poolboy
===> compile options: {erl_opts, [debug_info]}.
===> files to analyze ["/build/source/_checkouts/poolboy/src/poolboy.erl",
                              "/build/source/_checkouts/poolboy/src/poolboy_sup.erl",
                              "/build/source/_checkouts/poolboy/src/poolboy_worker.erl"]
===>      Compiled poolboy_worker.erl
===>      Compiled poolboy_sup.erl
===>      Compiled poolboy.erl
===> Compiling influx_udp
===> compile options: {erl_opts, [debug_info,warn_unused_vars,
                                         warnings_as_errors]}.
===> files to analyze ["/build/source/_checkouts/influx_udp/src/influx_udp_sup.erl",
                              "/build/source/_checkouts/influx_udp/src/influx_udp_worker.erl",
                              "/build/source/_checkouts/influx_udp/src/influx_udp.erl",
                              "/build/source/_checkouts/influx_udp/src/influx_udp_app.erl",
                              "/build/source/_checkouts/influx_udp/src/influx_line.erl"]
===>      Compiled influx_udp_sup.erl
===>      Compiled influx_udp_app.erl
===>      Compiled influx_udp.erl
===>      Compiled influx_udp_worker.erl
===>      Compiled influx_line.erl
===> Compiling yval
===> compile options: {erl_opts, [debug_info,warn_export_all,
                                         warn_export_vars,warn_missing_spec,
                                         warn_untyped_record,
                                         warn_unused_record,warn_bif_clash,
                                         warn_obsolete_guard,warn_unused_vars,
                                         warn_shadow_vars,warn_unused_import,
                                         warn_unused_function,
                                         warn_deprecated_function]}.
===> files to analyze ["/build/source/_checkouts/yval/src/yval.erl"]
===>      Compiled yval.erl
===> Compiling p1_utils
===> compile options: {erl_opts, [debug_info,{d,'HAVE_RAND'}]}.
===> files to analyze ["/build/source/_checkouts/p1_utils/src/p1_fsm.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_prof.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_options.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_shaper.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_nif_utils.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_utils.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_server.erl",
                              "/build/source/_checkouts/p1_utils/src/treap.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_proxy_protocol.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_queue.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_file_queue.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_rand.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_http.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_time_compat.erl",
                              "/build/source/_checkouts/p1_utils/src/p1_utils_sup.erl"]
===>      Compiled p1_server.erl
===>      Compiled p1_fsm.erl
===>      Compiled p1_utils_sup.erl
===>      Compiled p1_utils.erl
===>      Compiled p1_rand.erl
===>      Compiled p1_nif_utils.erl
===>      Compiled p1_shaper.erl
===>      Compiled p1_time_compat.erl
===>      Compiled p1_http.erl
===>      Compiled treap.erl
===>      Compiled p1_queue.erl
===>      Compiled p1_options.erl
===>      Compiled p1_prof.erl
===>      Compiled p1_proxy_protocol.erl
===>      Compiled p1_file_queue.erl
===> Compiling fast_tls
===> compile options: {erl_opts, [debug_info,
                                         {src_dirs,["src"]},
                                         {d,'REBAR3'}]}.
===> files to analyze ["/build/source/_checkouts/fast_tls/src/fast_tls.erl",
                              "/build/source/_checkouts/fast_tls/src/p1_sha.erl"]
===>      Compiled p1_sha.erl
===>      Compiled fast_tls.erl
===> Compiling stun
===> compile options: {erl_opts, [debug_info,
                                         {i,"include"},
                                         {d,'RAND_UNIFORM'}]}.
===> files to analyze ["/build/source/_checkouts/stun/src/stun_test.erl",
                              "/build/source/_checkouts/stun/src/stun_logger.erl",
                              "/build/source/_checkouts/stun/src/stun_shaper.erl",
                              "/build/source/_checkouts/stun/src/turn_tmp_sup.erl",
                              "/build/source/_checkouts/stun/src/stun_tmp_sup.erl",
                              "/build/source/_checkouts/stun/src/stun_sup.erl",
                              "/build/source/_checkouts/stun/src/stun_listener.erl",
                              "/build/source/_checkouts/stun/src/turn_sm.erl",
                              "/build/source/_checkouts/stun/src/stun_app.erl",
                              "/build/source/_checkouts/stun/src/stun_listener_sup.erl",
                              "/build/source/_checkouts/stun/src/stun.erl",
                              "/build/source/_checkouts/stun/src/stun_acceptor_sup.erl",
                              "/build/source/_checkouts/stun/src/stun_acceptor.erl",
                              "/build/source/_checkouts/stun/src/stun_codec.erl",
                              "/build/source/_checkouts/stun/src/turn.erl"]
===>      Compiled stun_sup.erl
===>      Compiled stun_tmp_sup.erl
===>      Compiled turn_tmp_sup.erl
===>      Compiled stun_acceptor_sup.erl
===>      Compiled stun_app.erl
===>      Compiled stun_listener_sup.erl
===>      Compiled turn_sm.erl
===>      Compiled stun_shaper.erl
===>      Compiled stun_logger.erl
===>      Compiled stun_listener.erl
===>      Compiled stun_acceptor.erl
===>      Compiled stun_codec.erl
===>      Compiled stun_test.erl
===>      Compiled turn.erl
===>      Compiled stun.erl
===> Compiling fast_yaml
===> compile options: {erl_opts, [debug_info,{src_dirs,["src"]}]}.
===> files to analyze ["/build/source/_checkouts/fast_yaml/src/fast_yaml_sup.erl",
                              "/build/source/_checkouts/fast_yaml/src/fast_yaml_app.erl",
                              "/build/source/_checkouts/fast_yaml/src/fast_yaml.erl"]
===>      Compiled fast_yaml_sup.erl
===>      Compiled fast_yaml_app.erl
===>      Compiled fast_yaml.erl
===> Compiling conf
===> compile options: {erl_opts, [debug_info,warn_export_all,
                                         warn_export_vars,warn_missing_spec,
                                         warn_untyped_record,
                                         warn_unused_record,warn_bif_clash,
                                         warn_obsolete_guard,warn_unused_vars,
                                         warn_shadow_vars,warn_unused_import,
                                         warn_unused_function,
                                         warn_deprecated_function]}.
===> files to analyze ["/build/source/_checkouts/conf/src/conf.erl",
                              "/build/source/_checkouts/conf/src/conf_yaml_backend.erl",
                              "/build/source/_checkouts/conf/src/odbc_yaml.erl",
                              "/build/source/_checkouts/conf/src/crypto_yaml.erl",
                              "/build/source/_checkouts/conf/src/kernel_yaml.erl",
                              "/build/source/_checkouts/conf/src/mnesia_yaml.erl",
                              "/build/source/_checkouts/conf/src/runtime_tools_yaml.erl",
                              "/build/source/_checkouts/conf/src/conf_backend.erl",
                              "/build/source/_checkouts/conf/src/conf_file.erl",
                              "/build/source/_checkouts/conf/src/conf_misc.erl",
                              "/build/source/_checkouts/conf/src/sasl_yaml.erl",
                              "/build/source/_checkouts/conf/src/conf_sup.erl",
                              "/build/source/_checkouts/conf/src/conf_env.erl",
                              "/build/source/_checkouts/conf/src/ssl_yaml.erl",
                              "/build/source/_checkouts/conf/src/stdlib_yaml.erl",
                              "/build/source/_checkouts/conf/src/os_mon_yaml.erl",
                              "/build/source/_checkouts/conf/src/conf_http.erl"]
===>      Compiled conf.erl
===>      Compiled conf_backend.erl
===>      Compiled conf_sup.erl
===>      Compiled runtime_tools_yaml.erl
===>      Compiled odbc_yaml.erl
===>      Compiled crypto_yaml.erl
===>      Compiled stdlib_yaml.erl
===>      Compiled sasl_yaml.erl
===>      Compiled ssl_yaml.erl
===>      Compiled os_mon_yaml.erl
===>      Compiled conf_env.erl
===>      Compiled mnesia_yaml.erl
===>      Compiled kernel_yaml.erl
===>      Compiled conf_file.erl
===>      Compiled conf_misc.erl
===>      Compiled conf_yaml_backend.erl
===>      Compiled conf_http.erl
===> Running hooks for erlc_compile in app recon (/build/source/_checkouts/recon) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app quantile_estimator (/build/source/_checkouts/quantile_estimator) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app prometheus (/build/source/_checkouts/prometheus) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app accept (/build/source/_checkouts/accept) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app prometheus_httpd (/build/source/_checkouts/prometheus_httpd) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app ulitos (/build/source/_checkouts/ulitos) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app poolboy (/build/source/_checkouts/poolboy) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app influx_udp (/build/source/_checkouts/influx_udp) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app yval (/build/source/_checkouts/yval) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app p1_utils (/build/source/_checkouts/p1_utils) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app fast_tls (/build/source/_checkouts/fast_tls) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app stun (/build/source/_checkouts/stun) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app fast_yaml (/build/source/_checkouts/fast_yaml) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for erlc_compile in app conf (/build/source/_checkouts/conf) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app recon (/build/source/_checkouts/recon) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app quantile_estimator (/build/source/_checkouts/quantile_estimator) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app prometheus (/build/source/_checkouts/prometheus) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app accept (/build/source/_checkouts/accept) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app prometheus_httpd (/build/source/_checkouts/prometheus_httpd) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app ulitos (/build/source/_checkouts/ulitos) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app poolboy (/build/source/_checkouts/poolboy) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app influx_udp (/build/source/_checkouts/influx_udp) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app yval (/build/source/_checkouts/yval) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app p1_utils (/build/source/_checkouts/p1_utils) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app fast_tls (/build/source/_checkouts/fast_tls) with configuration:
===> 	{provider_hooks, []}.
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app stun (/build/source/_checkouts/stun) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app fast_yaml (/build/source/_checkouts/fast_yaml) with configuration:
===> 	{provider_hooks, []}.
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app conf (/build/source/_checkouts/conf) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app recon (/build/source/_checkouts/recon) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app quantile_estimator (/build/source/_checkouts/quantile_estimator) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app prometheus (/build/source/_checkouts/prometheus) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app accept (/build/source/_checkouts/accept) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app prometheus_httpd (/build/source/_checkouts/prometheus_httpd) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app ulitos (/build/source/_checkouts/ulitos) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app poolboy (/build/source/_checkouts/poolboy) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app influx_udp (/build/source/_checkouts/influx_udp) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app yval (/build/source/_checkouts/yval) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app p1_utils (/build/source/_checkouts/p1_utils) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app fast_tls (/build/source/_checkouts/fast_tls) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app stun (/build/source/_checkouts/stun) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app fast_yaml (/build/source/_checkouts/fast_yaml) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app conf (/build/source/_checkouts/conf) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app recon (/build/source/_checkouts/recon) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app quantile_estimator (/build/source/_checkouts/quantile_estimator) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app prometheus (/build/source/_checkouts/prometheus) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app accept (/build/source/_checkouts/accept) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app prometheus_httpd (/build/source/_checkouts/prometheus_httpd) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app ulitos (/build/source/_checkouts/ulitos) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app poolboy (/build/source/_checkouts/poolboy) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app influx_udp (/build/source/_checkouts/influx_udp) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app yval (/build/source/_checkouts/yval) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app p1_utils (/build/source/_checkouts/p1_utils) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app fast_tls (/build/source/_checkouts/fast_tls) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app stun (/build/source/_checkouts/stun) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app fast_yaml (/build/source/_checkouts/fast_yaml) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app conf (/build/source/_checkouts/conf) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile with configuration:
===> 	{pre_hooks, []}.
===> Compile (project_apps)
===> Running hooks for compile in app eturnal (/build/source) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for erlc_compile in app eturnal (/build/source) with configuration:
===> 	{pre_hooks, []}.
===> Analyzing applications...
===> Compiling eturnal
===> compile options: {erl_opts, [warn_export_vars,
                                         warn_missing_spec_all,
                                         warn_unused_import,
                                         warnings_as_errors,deterministic]}.
===> files to analyze ["/build/source/src/eturnal_misc.erl",
                              "/build/source/src/mod_log_stun.erl",
                              "/build/source/src/eturnal_module.erl",
                              "/build/source/src/eturnal_yaml.erl",
                              "/build/source/src/mod_stats_prometheus.erl",
                              "/build/source/src/eturnal_logger.erl",
                              "/build/source/src/eturnal_sup.erl",
                              "/build/source/src/eturnal_systemd.erl",
                              "/build/source/src/mod_stats_influx.erl",
                              "/build/source/src/eturnal_app.erl",
                              "/build/source/src/eturnal_cert.erl",
                              "/build/source/src/eturnal.erl",
                              "/build/source/src/eturnal_ctl.erl"]
===>      Compiled eturnal_module.erl
===>      Compiled eturnal_sup.erl
===>      Compiled eturnal_app.erl
===>      Compiled mod_log_stun.erl
===>      Compiled mod_stats_influx.erl
===>      Compiled eturnal_misc.erl
===>      Compiled eturnal_logger.erl
===>      Compiled eturnal_cert.erl
===>      Compiled mod_stats_prometheus.erl
===>      Compiled eturnal_systemd.erl
===>      Compiled eturnal_yaml.erl
===>      Compiled eturnal_ctl.erl
===>      Compiled eturnal.erl
===> Running hooks for erlc_compile in app eturnal (/build/source) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for app_compile in app eturnal (/build/source) with configuration:
===> 	{pre_hooks, []}.
===> Running hooks for app_compile in app eturnal (/build/source) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile in app eturnal (/build/source) with configuration:
===> 	{post_hooks, []}.
===> Running hooks for compile with configuration:
===> 	{post_hooks, []}.
===> Running provider: release
===> Configuring releases the {relx, ...} entry from rebar.config
===> Running app-specific hooks
===> Running hooks for release in app eturnal (/build/source) with configuration:
===> 	{pre_hooks, [{release,"escript client/build client/stun"},
                             {release,"chmod o-rw config/eturnal.yml"}]}.
===> Running project-wide hooks
===> Running hooks for release with configuration:
===> 	{pre_hooks, []}.
===> Solving Release eturnal-1.12.0
===> Resolved eturnal-1.12.0
===> release: eturnal-1.12.0
     erts: 13.2.2.3
     goals:
          eturnal
          sasl
          recon
          influx_udp
          poolboy
          ulitos
          prometheus_httpd
          prometheus
          accept
          inets
          ssl
     applications:
          {kernel,"8.5.4.2"}
          {stdlib,"4.3.1.2"}
          {crypto,"5.1.4.1"}
          {p1_utils,"1.0.25"}
          {fast_yaml,"1.0.36"}
          {yval,"1.0.10"}
          {conf,"0.2.6"}
          {fast_tls,"1.1.16"}
          {stun,"1.2.9"}
          {asn1,"5.0.21"}
          {public_key,"1.13.3.1"}
          {eturnal,"1.12.0"}
          {sasl,"4.2"}
          {recon,"2.5.4"}
          {ulitos,"0.4.0",none}
          {poolboy,"1.5.2",none}
          {influx_udp,"1.1.2",none}
          {inets,"8.3.1.2",none}
          {prometheus,"4.10.0",none}
          {accept,"0.3.5",none}
          {prometheus_httpd,"2.1.11",none}
          {ssl,"10.9.1.2",none}

===> Assembling release eturnal-1.12.0...
===> Release output dir /build/source/_build/prod/rel/eturnal
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/kernel-8.5.4.2/ebin/kernel.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/stdlib-4.3.1.2/ebin/stdlib.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/crypto-5.1.4.1/ebin/crypto.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/p1_utils-1.0.25/ebin/p1_utils.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/fast_yaml-1.0.36/ebin/fast_yaml.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/yval-1.0.10/ebin/yval.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/conf-0.2.6/ebin/conf.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/fast_tls-1.1.16/ebin/fast_tls.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/stun-1.2.9/ebin/stun.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/asn1-5.0.21/ebin/asn1.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/public_key-1.13.3.1/ebin/public_key.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/eturnal-1.12.0/ebin/eturnal.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/sasl-4.2/ebin/sasl.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/recon-2.5.4/ebin/recon.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/ulitos-0.4.0/ebin/ulitos.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/poolboy-1.5.2/ebin/poolboy.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/influx_udp-1.1.2/ebin/influx_udp.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/inets-8.3.1.2/ebin/inets.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/prometheus-4.10.0/ebin/prometheus.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/accept-0.3.5/ebin/accept.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/prometheus_httpd-2.1.11/ebin/prometheus_httpd.app
===> Rewriting .app file: /build/source/_build/prod/rel/eturnal/lib/ssl-10.9.1.2/ebin/ssl.app
===> rendering builtin_hook_wait_for_process hook to _build/prod/rel/eturnal/bin/hooks/builtin/wait_for_process
===> Including Erts from /nix/store/q3rg8jf7p5azswqgwqmzxd7zx0zygzzb-erlang-25.3.2.6/lib/erlang
===> release start script created
===> Release successfully assembled: _build/prod/rel/eturnal
===> Stripping release beam files
===> Running app-specific hooks
===> Running hooks for release in app eturnal (/build/source) with configuration:
===> 	{post_hooks, []}.
===> Running project-wide hooks
===> Running hooks for release with configuration:
===> 	{post_hooks, []}.
@nix { "action": "setPhase", "phase": "installPhase" }
installing
ERTS found in /nix/store/8230bn8m34ns4mjz88c8qq9la0q2k9gp-eturnal-1.12.0/rel/eturnal/erts-13.2.2.3 - removing references to erlang to reduce closure size
consumeEntire(): ERROR: Input null bytes, won't process
/nix/store/3pfjacvdg9f491lfqc9qb2d0nknx73fb-stdenv-linux/setup: line 144: pop_var_context: head of shell_variables not a function context
```

</details>

### Steps To Reproduce
Steps to reproduce the behavior:
1. try to build the provided derivation

### Expected behavior
The package builds without issue, similarly to the following patched version:

<details>
<summary>Patched version that builds correctly</summary>

```nix
{ lib, beamPackages, fetchFromGitHub, libyaml }:
beamPackages.rebar3Relx rec {
  pname = "eturnal";
  version = "1.12.0";

  src = fetchFromGitHub {
    owner = "processone";
    repo = "eturnal";
    rev = version;
    sha256 = "BcFFsdPJ/zxs0iPvfiS0P801BqdxveL0hayc2Yld6qs=";
  };

  checkouts = beamPackages.fetchRebar3Deps {
    inherit version src;
    name = pname;
    sha256 = "stPusgmwNFRp2n4W6VehTmEejtgOWNblH3Z9nHQ0o8c=";
  };

  buildInputs = [ libyaml.dev ];
  buildPlugins = [ beamPackages.pc ];
  profile = "prod";
  releaseType = "release";

  # patch due to nixpkgs issue with release mode, just removes the postInstall hook
  installPhase = ''
    runHook preInstall
    dir=rel
    mkdir -p "$out/$dir" "$out/bin"
    cp -R --preserve=mode "_build/prod/$dir" "$out"
    find $out/rel/*/bin -type f -executable -exec ln -s -t $out/bin {} \;
  '';

  meta = with lib; {
    homepage = "https://eturnal.net/";
    description = "standalone STUN / TURN server";
    license = licenses.asl20;
    platforms = platforms.all;
  };
}
```

</details>

### Additional context
I was attempting to package the [eturnal stun/turn server](https://eturnal.net), but otherwise have never touched erlang before, so I may be using the builder wrong, however I haven't found any other packages using `releaseType = "release"` so I'm assuming its an issue with the builder. The profile and release type were detemined by reading eturnal's CI workflows.

The offending section of code: (pr #124779)
https://github.com/NixOS/nixpkgs/blob/68cf34ed01d8ccd9e8cad6be56efd74e1bab537b/pkgs/development/beam-modules/rebar3-release.nix#L87-L94

### Notify maintainers
@dlesl @happysalada (#124779)

Did not see anyone else who was obviously a maintainer.

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
admin@msi:~/ > nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.57, NixOS, 23.11 (Tapir), 23.11.20231011.5e4c2ad`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.17.0`
 - nixpkgs: `/etc/nix/inputs/nixpkgs`
```

--- a/pkgs/development/beam-modules/rebar3-release.nix
+++ b/pkgs/development/beam-modules/rebar3-release.nix
@@ -84,6 +84,7 @@ let
         runHook postInstall
       '';
 
+      # Release will generate a binary which will cause a read null byte failure, see #261354
       postInstall = lib.optionalString (releaseType == "escript") ''
         for dir in $out/rel/*/erts-*; do
           echo "ERTS found in $dir - removing references to erlang to reduce closure size"

--- a/pkgs/development/beam-modules/rebar3-release.nix
+++ b/pkgs/development/beam-modules/rebar3-release.nix
@@ -84,7 +84,7 @@ let
         runHook postInstall
       '';
 
-      postInstall = ''
+      postInstall = lib.optionalString (releaseType == "escript") ''
         for dir in $out/rel/*/erts-*; do
           echo "ERTS found in $dir - removing references to erlang to reduce closure size"
           for f in $dir/bin/{erl,start}; do

